### PR TITLE
Bugfix jsondecodeerror

### PR DESCRIPTION
**Pull request by taspinar**
As indicated in issue https://github.com/taspinar/twitterscraper/issues/93, sometimes a JSONDecodeError is returned by the requests library whenever its internal JSON decoder is called upon the response and the response is non existent (status code 500).
By first checking the status code of the response and only acting when the status code is 200, this error should not appear any more.

### CODE REVIEW

File: twitterscraper/query.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def query_single_page(url, html_response=True, retry=10):
 
     try:
         response = requests.get(url, headers=headers)
-        if resonse.status_code == 200:
+        if response.status_code == 200:
```
Comment (sils) [inline twitterscraper/query.py]: guess you made a typo in the first commit and corrected it in the second? I'd either `reset` back to master and make two clean commits or at least squash them, this makes the history rather confusing.

File: changelog.txt [hunk 2]
```diff
@@ -3,11 +3,14 @@
 ## 0.x.x
 
 TBD
+## 0.6.1 ( 2018-02-25 )
+### Fixed
+- Fixed a bug consisting of a call to requests internal JSON decoder when the response is None.
```
Comment (sils) [inline changelog.txt]: mentioning the issues we fix will probably help the reader to find out more

File: HISTORY.rst [hunk 3]
```diff
@@ -3,8 +3,13 @@
 ## 0.x.x
 
 TBD
+## 0.6.1 ( 2018-02-25 )
+### Fixed
+- Fixed a bug consisting of a call to requests internal JSON decoder when the response is None.
```
Comment (sils) [inline HISTORY.rst]: why do we have HISTORY *and* changelog?

File: twitterscraper/query.py [hunk 4]
```diff
@@ -32,21 +32,20 @@ def query_single_page(url, html_response=True, retry=10):
...
+            if html_response:
+                html = response.text
+                tweets = list(Tweet.from_html(html))
+                return tweets, "TWEET-{}-{}".format(tweets[-1].id, tweets[0].id)
+            else:
+                json_resp = response.json()
+                html = json_resp['items_html']
+                tweets = list(Tweet.from_html(html))
+                return tweets, json_resp['min_position']
         else:
```
Comment (sils) [inline twitterscraper/query.py]: we should have handling for non 200 error codes IMO, I usually find failing silently with no results letting the user believe everything went well is rather dangerous

**Comment by jeyoon** [comment]
Why did this branch end up being closed?

**Comment by taspinar** [comment]
@jeyoon I had implemented flawed logic and it was easier to implement the correct logic in a [new branch](https://github.com/taspinar/twitterscraper/tree/jsondecodeerror_fix) instead of reverting several commits.

**Pull request closed without merging**